ui/assets/css/_requirements.scss

```diff
@@ -1,7 +1,22 @@
 .req {
-  border: solid 1px #d6d7d9;
-  margin: 10px 0;
-  list-style: none;
-  padding: 10px 15px;
-  width: 700px;
+  border-color: $color-gray-light;
+
+  .req-id {
+    color: $color-gray;
+    font-family: "Merriweather", serif;
+  }
+
+  .req-text {
+    color: $color-gray-dark;
+    font-family: "Merriweather", serif;
+    font-weight: 300;
```

Comment:
Not for this round, but does it make sense to put these font family, weight, size, etc. in a mixin? I assume we'll want to reuse for consistency.